notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -7862,6 +7862,7 @@ anti-technology way of life [The inventor of virtual reality Dr. Benjamin Trace,
 data privacy [The virtual reality entrepreneur Jonathan Walker used his technology to clandestinely look up "all the contributors to fundamental religions over the past five years" and "all hypocritical Democrats making over 300,000 a year".]
 brother and sister [The runaway teens Travis and Jade were siblings.]
 looting [People were shown looting shops after Jobe crashed Los Angeles' various computer systems.]
+false friend [Jobe manipulated his old friend the now 16-year-old Peter into giving him information about a hidden Nano routine in the Chyron chip's design.]
```

Comment:
Did we not decide to use this theme exclusively when the motive is to cause harm or trouble, as opposed for personal gain? Comment sounds like it was only for personal gain.

Reply:
Same problem here. There's no usage of **false fried** in the master branch version.

Reply:
Ok I get that master is already fixed somehow, but doesn't that mean we ought to close this pull request without merging it? It seems wrong that I should approve and merge a diff that shows me errors...